internal/app/machined/pkg/runtime/v1alpha1/platform/platform.go

```diff
@@ -67,6 +68,9 @@ func CurrentPlatform() (p runtime.Platform, err error) {
 		platform = *p
 	}
 
+	flag.StringVar(&platform, "platform", "", "the platform")
+	flag.Parse()
```

Comment:
Add the ability to supply a flag to `machined` specifying the platform. It makes using the Talos container feel cleaner.